plugins/tiff/src/tiffpresentation.cc

```diff
@@ -389,6 +401,85 @@ void TiffPresentation::done()
   tif = NULL;
 }
 
+/**
+ * Add two pipette color map values of the same key.
+ * 
+ * Loops through rhs, adds values of the same key to each other and assigns it back to lhs.
+ * Return lhs at the end.
+ */
+PipetteLayerOperations::PipetteColor sumPipetteColors(const PipetteLayerOperations::PipetteColor& lhs, const PipetteLayerOperations::PipetteColor& rhs)
+{
+  PipetteLayerOperations::PipetteColor result;
+  for(auto const& elem : rhs)
+  {
+    result[elem.first] = elem.second;
+    if(lhs.find(elem.first) != lhs.end())
+    {
+      result[elem.first] += lhs.at(elem.first);
```

Comment:
... so first you copy `rhs` into result (in a roundabout way), and then, if `lhs` shares any key with `rhs`, you sum those values, but any key in `lhs` that is not shared by `rhs` is ignored.

That's quite different from what it did before. Why the change of heart?

Reply:
The problem for me was that if `lhs` is a `const`, we can not access elements using `[]` i.e. `lhs[element]` but have to use `.at()`. The issue here is then that this would throw an error when the key is not in the map, which is different to before where it would default to `0` if you used `[]` and the key was not in the map.

Reply:
But still. Why not
```
result  = lhs;
for(auto& [k,v] : rhs)
  result[k] = v;
```
or something similar?

Reply:
I have not thought about this 😅 . Note that this does throw a warning:
`warning: decomposition declaration only available with -std=c++1z or -std=gnu++1z`. I guess the `[k,v]` can be replaced by `elem`, and we can access the key and value using `elem.first` and `elem.second` respectively, like we did before.